### PR TITLE
Revert "Revert "MGMT-5430 Move assisted service image to stream8""

### PR DESCRIPTION
**Pull request by flaper87**
This is the round two! Let's make all the required changes now in both, this repo and CI workflows.

Reverts openshift/assisted-service#1774

### CODE REVIEW

**Comment by lranjbar** [comment]
/hold 

The images seem to be building but it is still replacing the stream8 buildroot improperly.

**Comment by lranjbar** [comment]
@flaper87 This one looks good now!

/lgtm

File: Dockerfile.assisted-service [hunk 1]
```diff
@@ -25,31 +25,10 @@ RUN CGO_ENABLED=0 GOFLAGS="" GO111MODULE=on go build -o /build/assisted-service-
...
-    dnf install -y python3-nispor && \
-    dnf install -y NetworkManager && \
-    dnf install -y NetworkManager-ovs && \
-    dnf install -y NetworkManager-team && \
-    dnf install -y NetworkManager-config-server && \
-    dnf install -y openvswitch2.11 && \
-    dnf install -y python3-openvswitch2.11 && \
-    dnf install -y nmstate &&\
-    pip3 install pycairo && \
-    dnf remove -y dnf-plugins-core && \
```
Comment (ronniel1) [inline Dockerfile.assisted-service:52]: How is it that we don't need all of these anymore?

**Comment by YuviGold** [comment]
Let's make sure, though it shouldn't affect those jobs.

/test e2e-metal-assisted-single-node
/test e2e-metal-assisted-ipv6
/test system-test-olm

File: Dockerfile.assisted-service (hunk 1 above)
Comment (flaper87) [reply Dockerfile.assisted-service:52]: They were dependencies for the `nmstate` build, which I, mistakenly, didn't remove in https://github.com/openshift/assisted-service/pull/1585

**Pull request merged**